master/getting-started/bare-metal/bare-metal.md

```diff
@@ -427,3 +427,90 @@ an ingress rule allowing access *to* port 999 and an egress rule allowing
 is usually enough to specify the ingress rule only, and then connection
 tracking will automatically allow the return path.)
+
+## Pre-DNAT policy
+
+Policy for host endpoints can be marked as 'preDNAT'.  This means that rules in
+that policy should be applied before any DNAT (Destination Network Address
+Translation), which is useful if it is more convenient to specify Calico policy
+in terms of a packet's original destination IP address and port, than in terms
+of that packet's destination IP address and port after it has been DNAT'd.
+
+An example is securing access to Kubernetes NodePorts from outside the cluster.
+Traffic from outside is addressed to any node's IP address, on a known
+NodePort, and Kubernetes (kube-proxy) then DNATs that to the IP address of one
+of the pods that provides the corresponding Service, and the relevant port
+number on that port (which is usually different from the NodePort).
+
+As NodePorts are the externally advertised way of connecting to Services (and a
+NodePort uniquely identifies a Service, whereas an internal port number may
+not), it makes sense to express Calico policy to expose or secure particular
+Services in terms of the corresponding NodePorts.  But that is only possible if
+the Calico policy is applied before DNAT changes the NodePort to something
+else - and hence this kind of policy needs 'preDNAT' set to true.
+
+## When do host endpoint policies apply?
+
+As stated above, normal host endpoint policies apply to traffic that arrives on
+and/or is sent to a host interface, except if that traffic comes from or is
+destined for a workload on the same host.  The rules for applying untracked and
+pre-DNAT policies are different in some cases, so here we present all of those
+rules together, for all possible flows and all types of host endpoints policy.
+
+For packets that arrive on a host interface and are destined for a local
+workload - i.e. a locally-hosted pod, container or VM:
+
+- Normal policies do not apply - by design, because Calico enforces the
+  destination workload's ingress policy in that case.
+
+- Untracked policies do not apply - because there is no way to make them work
+  in both the forwards and reverse directions.
+
+  > **NOTE**
+  >
+  > To be precise, untracked policy for the incoming host interface may apply in
+  > the forwards direction, and if so it will have the effect of forwarding the
+  > packet to the workload without any connection tracking.  Then, in the reverse
+  > direction, there will be no conntrack state for the return packets to match,
+  > and there is no application of any outbound rules that may be defined by the
+  > untracked policy - so unless the workload's policy specifically allows the
+  > relevant source IP, the return packet will be dropped.  That is the same
+  > overall result as if there was no untracked policy at all, so in practice we
+  > can describe this as untracked policies not applying to this flow.
+
+- Pre-DNAT policies _do_ apply.
```

Comment:
Pre-DNAT policy doesn't know if it's going to a workload, because it's pre-routing.